Fails to build with LDC on Arch GNU/Linux

When I try to build this with LDC, I get the following:

```
[koz@Emi dub]$ ./build.sh
Generating version file...
Running ldmd2...
/usr/bin/ld: cannot find -lphobos2-ldc
collect2: error: ld returned 1 exit status
Error: /usr/bin/gcc failed with status: 1
```

What do I need to change to make this build with LDC? Is this even possible?

--- a/build.sh
+++ b/build.sh
@@ -16,7 +16,7 @@ if [ "$DMD" = "" ]; then
 	exit 1
 fi
 
-VERSION=$($DMD --version 2>/dev/null | sed -n 's|DMD.* v||p')
+VERSION=$($DMD --version 2>/dev/null | sed -En 's|.*DMD.* v([[:digit:]\.]+).*|\1|p')
 # workaround for link order issues with libcurl (phobos needs to come before curl)
 if [[ $VERSION < 2.069.0 ]]; then
 	# link against libcurl

